tox.ini

```diff
@@ -58,3 +58,11 @@ usedevelop = True
 deps = -r requirements/docs.txt
 commands =
     sphinx-build -W -b html . _build
+
+[testenv:release]
```

Comment:
This can be removed right?

Feel free to squash/merge after removing this, thanks!

Reply:
> This can be removed right?

Yes, I'll do it.

> Feel free to squash/merge after removing this, thanks!

I don't have write access to this repo :sweat:

Reply:
Just sent an invitation. 😉

Reply:
Thanks :tada: